workflows/prognostic_c48_run/examples/fine_res_ml.yml

```diff
@@ -0,0 +1,20 @@
+base_version: v0.5
```

Comment:
What's up with these "examples"? If they are only used for testing, I would bury them in the tests, since their purpose is not quite clear in the present location.

Reply:
Done